5.2.cup_of_join.py

```diff
@@ -0,0 +1,17 @@
+def join(*lists, sep='-'):
+    """Concatenates an arbitrary number of lists."""
+    if lists is None:
+        return None
+    result = []
+    for lst in lists:
```

Comment:
Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing